assets/src/styles/blocks/PageHeader.scss

```diff
@@ -28,6 +28,12 @@ $text-column-padding-in: 24px;
     margin: -$sp-5x $sp-3 $sp-3;
     padding-inline-start: 0;
     padding-inline-end: 0;
+    z-index: 1;
+
+    .wp-block-button,
+    .wp-block-button a {
```

Comment:
You can use `wp-block-button__link` instead.

Reply:
We could indeed, tbh I just copy-pasted this from the [Media & Text styles](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/master/assets/src/styles/blocks/core-overrides/_media-text.scss#L26-L30) since this is what these lines override